code/game/turfs/turf.dm

```diff
@@ -533,6 +533,9 @@ GLOBAL_LIST_INIT(unweedable_areas, typecacheof(list(
 			else
 				has_obstacle = TRUE
 				break
+		if(istype(O, /obj/effect/alien/weeds/node))
```

Comment:
This proc would also benefit from either a redefined proc to check the objects, or a flag.